Classes/ScrnM4203AssaultRifle.uc

```diff
@@ -8,8 +8,44 @@ var         name             ReloadShortAnim;
 var         float             ReloadShortRate;
 
 var transient bool  bShortReload;
+var vector ZoomedViewOffset; //to fix sight in RenderOverlays
+
+
+//copypaste to add additional offset
+simulated event RenderOverlays( Canvas Canvas )
+{
+	local int m;
+    local vector DrawOffset;
+	if (Instigator == None)
```

Comment:
Tabs instead of spaces... again :)

Reply:
I fixed it, seems its caused because I used webui to upload or edit this. Guess I can't use it anymore.